src/consumption.cpp

```diff
@@ -1470,7 +1470,11 @@ bool Character::fuel_bionic_with( item &it )
 
 int Character::get_acquirable_energy( const item &it ) const
 {
-    const bionic_id &bid = get_most_efficient_bionic( get_bionic_fueled_with( it ) );
+    std::vector<bionic_id> bids = get_bionic_fueled_with( it );
```

Comment:
```suggestion
   const std::vector<bionic_id> &bids = get_bionic_fueled_with( it );
```
I think this can also be a const